### ISSUE TITLE
alignment issue for team members

### ISSUE THREAD

**Issue by meisabhishekpatel** [ContriHUB/CodeSangam#1]
Well since the name of winner can be of any length so current podium's winner are not correctly aligned.
So the task is to fix that so team member are perfectly aligned as they should be.

![image](https://github.com/user-attachments/assets/8ce4d44b-9d66-4474-9f3c-f413cc831baf)

**Comment by waquar003** [comment]
@meisabhishekpatel plz assign this to me

**Comment by jatin31195** [comment]
please add readme file @meisabhishekpatel

**Comment by jatin31195** [comment]
Please add readme in order i setup the project

**Comment by meisabhishekpatel** [comment]
@jatin31195  if solve this without readme you bonus points

**Comment by meisabhishekpatel** [comment]
if you still can't do it ping me I'll create a readme for you.

**Comment by meisabhishekpatel** [comment]
@jatin31195 I am updating the readme so no bonus for you.

**Comment by jatin31195** [comment]
Unassign the project make go many things are used i dont even have knowlege
of these

**Comment by meisabhishekpatel** [comment]
@waquar003 do you want to work on this issue now ? If yes then request again on contrihub website

**Comment by jatin31195** [comment]
No, sorry.

**Comment by meisabhishekpatel** [comment]
@jatin31195 read the message and "issues" carefully.

**Comment by jatin31195** [comment]
Wait let me try again

**Comment by jatin31195** [comment]
ok i don't want to work on this remove this from my currently active issue

**Comment by meisabhishekpatel** [comment]
@jatin31195 get your mind straight. Okay

**Comment by Ayush5071** [comment]
@meisabhishekpatel Sir, I would like to work on this issue.

**Comment by meisabhishekpatel** [comment]
@Ayush5071 you are assigned this issue

**Comment by Ayush5071** [comment]
@meisabhishekpatel sir, do i have to take care of responsivenes also, or just in dektop .

**Comment by meisabhishekpatel** [comment]
@Ayush5071 yes

**Comment by meisabhishekpatel** [comment]
remember it's a bonus issue.

**Comment by Ayush5071** [comment]
Yes, sir I'll try, main thing is podium component file is big and complex 🙂
I'll do my best to get it resolved.

**Comment by Ayush5071** [comment]
@meisabhishekpatel sir,PR request send.

**Comment by Ayush5071** [comment]
@meisabhishekpatel sir , now the alignment of names are perfect and also responsive , check video in PR request.

**Comment by Ayush5071** [comment]
@meisabhishekpatel sir please, see the PR Request i have sended

**Comment by kaurpreetman** [comment]
My issue assignment request has been accepted. @meisabhishekpatel, could you please confirm if I should start working on this issue or if @Ayush5071 has already completed the task?

**Comment by meisabhishekpatel** [comment]
@kaurpreetman you can start too, @Ayush5071 has not solved the issue yet

**Comment by hexwhiz** [comment]
I have been assigned the issue. I will be working on it now.

**Comment by hexwhiz** [comment]
@meisabhishekpatel I have made a PR.

**Comment by meisabhishekpatel** [comment]
the issue closed now
